.github/workflows/website-validation.yml

```diff
@@ -4,6 +4,11 @@ on:
     types: [opened, synchronize, reopened]
     paths:
       - "docs/**"
+      - "*.md"
+      - "**/*.md"
+    paths-ignore:
+      - "build-tools/**"
+      - "server/**"
```

Comment:
Are you sure this works? From [these docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore):

> You cannot use both the paths and paths-ignore filters for the same event in a workflow. If you want to both include and exclude path patterns for a single event, use the paths filter prefixed with the ! character to indicate which paths should be excluded.

Reply:
Updated the path exclusion. Thanks for the correction!